### PR TITLE
mv theme folder to bidl-project.github.io and update readme.md 

### PR DESCRIPTION
**Pull request by helenlly**
## What changes were proposed in this pull request?
1) mv theme folder to bidl-project.github.io
2) update readme.md

### CODE REVIEW

**Comment by helenlly** [comment]
add folders as layer structure as below

optim:
	Metrics
nn: 
	Containers
	Convolution Layers
	Loss Functions
	Distance Functions
	Dropout Layers
	Embedding Layers
	Merge Layers
	Non-linear Activations
	Normalization Layers
	Pooling Layers
	Recurrent Layers
	Simple Layers
	Utilities
	Functional API

File: docs/README.md [hunk 1]
```diff
@@ -5,10 +5,14 @@ All official released docs are available in [BigDL Docs](https://bigdl-project.g
...
-## Building the documentation in local server:
+## Building the documentation:
 
-- install MkDocs: `pip install mkdocs`
-- `cd` to the `docs/'folder and run:
-    - `mkdocs serve`    # Starts a local webserver and you can view in:  [localhost:8000](localhost:8000) , e.g. http://127.0.0.0:8000
-    
-    - `mkdocs build`    # Builds a static site in "site" directory, you can view site/index.html directly
+1. install MkDocs: `pip install mkdocs`
+2. `cp` the folder 'readthedocs/' from https://github.com/helenlly/bigdl-project.github.io to "docs/"
```
Comment (yiheng) [inline docs/README.md]: how do you cp a folder from a website?

Comment (helenlly) [inline docs/README.md]: download?

**Comment by helenlly** [comment]
will add a script to download and build the docs soon; temporally need the user download the theme folder byself

**Comment by jason-dai** [comment]
No need for a script; manual download is fine.

**Comment by yiheng** [comment]
I think just two commands are okay.

wget https://github.com/helenlly/bigdl-project.github.io/archive/master.zip
unzip master.zip -d docs/

**Pull request merged**